task/sql-tasks.js

```diff
@@ -129,7 +195,14 @@ async function task_1_8(db) {
  *
  */
 async function task_1_9(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            CustomerID,
+            ContactName
+        FROM Customers
+        WHERE ContactName REGEXP '^[F][^\.]{2}[n]'

```

Comment:
Use regexp only if not possible to use **like**. In this case it's possible to replace it with like